docs/tutorials/01_neural_networks.ipynb

```diff
@@ -368,35 +440,71 @@
    "id": "olympic-coral",
    "metadata": {},
    "source": [
-    "As in the previous example, we don't set the `sampler` parameter, the network will create an instance of the reference `Sampler` primitive for us."
+    "### 4.2. Constructing SamplerQNN using the circuit\n",
+    "Now we define the `SamplerQNN` by providing the parameterized circuit, input and weight as parameters. We don't set the `sampler` parameter, therefore the network will create an instance of the reference `Sampler` primitive for us."
```

Comment:
While the example for `SamplerQNN` is not great, I think it is essential, if we keep this example, to explain why `nput_params=[]`.

Reply:
I left a suggestion explaining why it is so.